bundles/org.openhab.binding.octopusenergy/README.md

```diff
@@ -0,0 +1,167 @@
+# Octopus Energy Binding
+
+This binding allows OpenHAB to communicate with the public API from Octopus Energy (https://octopus.energy), an energy provider in the UK.
+The following features are provided by the binding:
+
+- retrieval of latest meter readings (consumption)
+- retrieval of tariff schedule / unit rates (if on an agile tariff, half-hourly future rates are provided based on availability)
+- calculation of forward looking least cost window based on given energy consumption. I.e. when is the cheapest time to run my dishwasher? This is available as a single pair of channels (cheapestSlotDuration, cheapestSlotStart) and also as an action for the use in scripts/rules.
+
+#### Current Limitations / Work in Progress
+
+- only tested with Octopus "Agile" electricity tariff. I would welcome users of "Go" or "Go Faster" for testing.
+- doesn't do very much with historical data
+
+## Supported Things
+
+This binding supports the following thing types
+
+| Thing                 | Thing Type | Discovery | Description                                    |
+|-----------------------|------------|-----------|------------------------------------------------|
+| Bridge                | Bridge     | Manual    |  A single connection to the Octopus Energy API |
+| ElectricityMeterPoint | Thing      | Automatic |  An electricity meter point                    |
+| GasMeterPoint         | Thing      | Automatic |  An gas meter point                            |
```

Comment:
Your examples below seem to indicate this list is wrong with the capital at the start?

Reply:
done